reducers/images_reducer.js

```diff
@@ -56,7 +57,7 @@ export default function(state = INITIAL_STATE, action) {
                                     type: action.payload.type,
                                     selected: false,
                                     tags: {},
-                                    picked_up: false,
+                                    pickedUp: action.payload.pickedUp,
```

Comment:
keep `picked_up` everywhere

Reply:
we have used `camelCase` for all the variables declared on frontend.